### PR TITLE
IntuneDeviceConfigurationPolicyAndroidOpenSourceProject: Initial release

### PR DESCRIPTION
**Pull request by William-Francillette**
#### Pull Request (PR) description

Initial release

### CODE REVIEW

**Comment by andikrueger** [comment]
Can you please add a description of this change to the Changelog, under a Unreleased section at the top of the file?

File: Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationPolicyAndroidOpenSourceProject/MSFT_IntuneDeviceConfigurationPolicyAndroidOpenSourceProject.psm1 [hunk 1]
```diff
@@ -0,0 +1,1539 @@
...
+        if($null -eq $context)
+        {
+            $ConnectionMode = New-M365DSCConnection -Workload 'MicrosoftGraph' `
+                -InboundParameters $PSBoundParameters -ProfileName 'beta'
+        }
+        Select-MgProfile 'beta' -ErrorAction Stop
+    }
+    catch
+    {
+        Write-Verbose -Message "Reloading1"
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationPolicyAndroidOpenSourceProject/MSFT_IntuneDeviceConfigurationPolicyAndroidOpenSourceProject.psm1:125]: Please remove testing verbose

**Review by NikCharlebois — CHANGES_REQUESTED** [review]

**Review by NikCharlebois — APPROVED** [review]

**Comment by NikCharlebois** [comment]
@William-Francillette can you looks at fixing the unit tests? Thanks

**Comment by William-Francillette** [comment]
> @William-Francillette can you look at fixing the unit tests? Thanks

@NikCharlebois Not sure the issue, but pester succeed on my side
image

The test unit is failing for another resource
image

**Pull request merged**